include/openssl/x509v3.h

```diff
@@ -350,7 +350,7 @@ struct ISSUING_DIST_POINT_st {
 # define EXFLAG_BCONS            0x1
 # define EXFLAG_KUSAGE           0x2
 # define EXFLAG_XKUSAGE          0x4
-# define EXFLAG_NSCERT           0x8
+# define EXFLAG_NSCERT           0x0
```

Comment:
Add a note to say that `its old value should not reused` ?

Reply:
done.